src/click/termui.py

```diff
@@ -153,7 +153,10 @@ def prompt_func(text):
         try:
             result = value_proc(value)
         except UsageError as e:
-            echo(f"Error: {e.message}", err=err)  # noqa: B306
+            if hide_input:
```

Comment:
I could have probably used `.format` to use conditional checking on `hide_input` to `echo` on a single line, but I thought that wouldn't have been very readble for its length. Feedback welcome.